instance/models/instance.py

```diff
@@ -495,7 +529,14 @@ class OpenEdXInstance(AnsibleInstanceMixin, GitHubInstanceMixin, Instance):
     s3_secret_access_key = models.CharField(max_length=50, blank=True)
     s3_bucket_name = models.CharField(max_length=50, blank=True)
 
-    ANSIBLE_SETTINGS = AnsibleInstanceMixin.ANSIBLE_SETTINGS + ['ansible_s3_settings']
+    database_url = models.CharField(max_length=255, blank=True, default=settings.INSTANCE_DATABASE_URL or '')
```

Comment:
Using an application setting as the default causes migrations to be generated whenever the current setting differs. See https://github.com/open-craft/opencraft/pull/7#discussion_r40174892